README.md

```diff
@@ -33,13 +34,26 @@ init_commands: []
 log_level: info
 ```
 
-If you are running AppDaemon in your own docker container, you must create and build a docker image with the above dependencies using a [Dockerfile](https://docs.docker.com/engine/reference/builder/) similar to this (**untested**):
+### AppDaemon 4 in Docker
 
-```Dockerfile
-FROM acockburn/appdaemon:latest
-# Manually install the dependencies
-RUN apk add py3-pillow
-RUN pip3 install Pillow
+You will need to add the following [dependencies](https://appdaemon.readthedocs.io/en/latest/DOCKER_TUTORIAL.html/):
+
+#### system_packages.txt
+```
+py3-pillow
+musl
+make
+g++
+python3-dev
+build-base
+jpeg-dev
+zlib-dev
```

Comment:
These seem legitimate to me.

@desertblade could you try again building it without these?:
`system_packages.txt`
```diff
-jpeg-dev
-zlib-dev
```

`requirements.txt`
```diff
-image
```